e2e/multi-app-project/multi_app_project_test.go

```diff
@@ -146,32 +146,19 @@ var _ = Describe("Multiple App Project", func() {
 					AppName:     appName,
 				})
 				Expect(appShowErr).NotTo(HaveOccurred())
-				Expect(len(app.Routes)).To(Equal(1))
 
-				// Call each environment's endpoint and ensure it returns a 200
-				route := app.Routes[0]
-				Expect(route.Environment).To(Equal("test"))
-				// Since the front end was added first, it should have no suffix.
+				Expect(len(app.ServiceDiscoveries)).To(Equal(1))
+				Expect(app.ServiceDiscoveries[0].Environment).To(Equal([]string{"test"}))
+				Expect(app.ServiceDiscoveries[0].Namespace).To(Equal(fmt.Sprintf("%s.%s.local:80", appName, projectName)))
+
 				if appName == "front-end" {
+					// Load balanced web apps have routes.
+					Expect(len(app.Routes)).To(Equal(1))
+					route := app.Routes[0]
+					Expect(route.Environment).To(Equal("test"))
```

Comment:
Should we try to ping the url here? http.Get(route.URL)

Reply:
I'll consolidate both yours and David's comment above and add a third application where the two lb web apps will have their URLs tested.

Reply:
Done!